### DISCUSSION TITLE
New IKEA STYRBAR remote - can it be supported?

### DISCUSSION THREAD

**Discussion by ikac2000** [Koenkk/zigbee2mqtt#6724] [Devices]
Hello community,

Ikea introduced a nece new controller, Styrbar, that works on 2 AAA batteries.

Is there a possibility for it to be supported out-of-the-box for zigbee2mqtt?

Thanks!

**Comment by kpmkpm** [comment]
Bought also that one. Anybody already working with the support?

**Comment by sjorge** [comment]
Probably won’t be to hard to support, I haven’t seen any of the usually people that add device have one. So probably nobody has gotten around to it.

having a database.db entry for one is probably a good start. If they are similar enough one can probably take an educated guess and have a devices.js entry to test.
